### DISCUSSION TITLE
How do I change the time plugin to 12hr format?

### DISCUSSION THREAD

**Discussion by shndowbots** [FelixKratz/SketchyBar#264] [Q&A] [answered]
How do I change the time plugin to 12hr?
Here is the sh file.

Thank you!
```
#!/usr/bin/env sh

sketchybar --set $NAME icon="$(date '+%a %d. %b')" label="$(date '+%H:%M')"

```

**Comment by FelixKratz** [comment, marked as answer]
The time is provided by the command `date`. What you want is to find the argument which gives you 12 hour time format. To do this you use:
```bash
man date
```
What you will find is that `%H` is the hour in 24h format and `%I` the hour using 12h format. Additionally you will find that `%p` gives `AM` or `PM` depending on the current time. The command you are looking for is thus:
```bash
date '+%I:%M %p'
```
Replace the command invoked for the clock label with this one and you are good to go.

**Reply by shndowbots** [reply]
Works perfectly, thanks!

**Comment by exact** [comment]
this is the most baffling project/discussion to me, what the hell is even the name of the property to set
